sys-libs/libatomic_ops.py

```diff
@@ -0,0 +1,30 @@
+import stdlib
+from stdlib.template import autotools
+from stdlib.template.configure import configure
+from stdlib.manifest import manifest
+
+
+@manifest(
+    name='libatomic-ops',
+    category='sys-libs',
```

Comment:
```suggestion
    category='dev-libs',
```

dev-libs is more appropriate.